css/css-color/t425-hsla-basic-a.xht

```diff
@@ -5,11 +5,11 @@
 		<link rel="author" title="L. David Baron" href="https://dbaron.org/" />
 		<link rel="author" title="Mozilla Corporation" href="http://mozilla.com/" />
 		<link rel="help" href="http://www.w3.org/TR/css3-color/#hsla-color" />
+		<link rel="match" href="t425-hsla-basic-a-ref.html" />
 		<meta name="flags" content="" />
 		<meta name="assert" content="Test basic functioning of hsla() colors." />
 		<style type="text/css"><![CDATA[
 		html, body { background: white; }
-		p { font-weight: bold; margin: 0; }
```

Comment:
No, leave the bold in there. It is easier to see the correct color on bolder text.